packages/block-library/src/navigation/index.php

```diff
@@ -323,6 +323,10 @@ function block_core_navigation_maybe_use_classic_menu_fallback() {
 			)
 		);
 
+		if ( is_wp_error( $wp_insert_post_result ) ) {
```

Comment:
To do this, you need to pass, second param of `wp_insert_post` as true.

Reply:
Thanks! Done